cms/grading/__init__.py

```diff
@@ -347,6 +350,22 @@ def format_status_text(status, translator=None):
                      "text: %r", status, exc_info=True)
         return translator("N/A")
 
+def AIOTranslator(translator):
+    def simplify_status_text(status):
+        mapping_startswith = {
+            "Execution failed because of sandbox error"             : "Judge error, please notify judges",
+            "Execution killed because of forbidden syscall"         : "Program killed due to illegal operation, double check rules regarding allowed system calls",
```

Comment:
"Program killed due to illegal operation. Please check the rules regarding allowable system calls."